### PR TITLE
fix(email-edit): text error

### PR DESCRIPTION
**Pull request by varun257**
### Description of the Change

change password confirmation label

### Benefits

Fixes text error

### Possible Drawbacks

None

### Applicable Issues

MFRWW-896

### CODE REVIEW

**Comment by jleveugle** [comment]
Hello @varun257,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Comment by varun257** [comment]
Needs to be merged to develop

File: src/emailpro/account/update/emailpro-account-update.html [hunk 1]
```diff
@@ -112,7 +112,7 @@
                      data-ng-class="{ 'has-error': selectedAccount.passwordConfirmation && differentPasswordFlag }">
                     <label for="passwordConfirmation" class="col-md-3 control-label"
                            data-ng-class="{ 'required': !selectedAccount.canBeConfigured }"
-                           data-translate="emailpro_ACTION_update_account_step1_password_confirmation_label"></label>
+                           data-translate="exchange_account_add_passwordConfirmation_label"></label>
```
Comment (antleblanc) [inline src/emailpro/account/update/emailpro-account-update.html]: I'm curious about the fact that you are here replacing the translation key with this new prefix `exchange_` whereas we are in the `emailpro` module.

Comment (varun257) [reply src/emailpro/account/update/emailpro-account-update.html]: It was cross referenced in manager-web i think. 
Have fixed it now. It was a problem of inconsistent translations across EN_GB and other English files. Dont know if i should update this or if this should go through translation team though!

**Comment by varun257** [comment]
Have removed `qtlid` for `fr` translation and removed all other modifications.

File: src/emailpro/translations/Messages_fr_FR.xml [hunk 2]
```diff
@@ -259,7 +259,7 @@
    <translation id="emailpro_ACTION_update_account_step1_display_name_label" qtlid="23515">Nom à afficher</translation>
    <translation id="emailpro_ACTION_update_account_step1_first_name_label" qtlid="23323">Prénom</translation>
    <translation id="emailpro_ACTION_update_account_step1_last_name_label" qtlid="3818">Nom</translation>
-   <translation id="emailpro_ACTION_update_account_step1_password_confirmation_label" qtlid="9983">Confirmation</translation>
+   <translation id="emailpro_ACTION_update_account_step1_password_confirmation_label">Confirmation</translation>
```
Comment (cbourgois) [inline src/emailpro/translations/Messages_fr_FR.xml]: Hello @varun257,

I'm not sure to understand what is the purpose of this change (and the only one changes in this PR) : 
If we remove a `qtlid` but have no change on the translation, the same `qtlid` will apply when we submit/retrieve translations, and this change will have no effect.

Comment (varun257) [reply src/emailpro/translations/Messages_fr_FR.xml]: Hi @cbourgois ,

The translation for this is not consistent across all files. In `en_US` it is `"Confirmation"` and in `en_CA/en_GB` it is `"Order Confirmation".`
The translation should be `"Confirmation"` in all english translations. Can you suggest how to make this consistent? Thanks

Comment (varun257) [reply src/emailpro/translations/Messages_fr_FR.xml]: And this issue seems to be breaking the UI as per the issue mentioned in Applicable issues.

**Comment by jleveugle** [comment]
Hello @varun257,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3
